code/modules/mob/living/silicon/robot/dogborg/dogborg.dm

```diff
@@ -60,7 +60,7 @@
 	modules += new /obj/item/weapon/reagent_containers/syringe(src) //In case the chemist is nice!
 	modules += new /obj/item/weapon/reagent_containers/glass/beaker(src)//For holding the chemicals when the chemist is nice
 	..()
-	emag = null
+	emag = null //suggestion?
```

Comment:
I think the null emag as it is was done to keep the mounted laser off? Sounds fair but if asking for a different emag module I can't think of any right now